src/mixins/events/index.js

```diff
@@ -10,8 +10,20 @@ export default {
      * @param {Event} event
      * @return {*|default.methods}
      */
-    propagate(event) {
+    propagateEvent(event) {
       return this.$emit(event.type, event);
     },
+
+    /**
+     * This method help propagate event values to
+     * parents components in vue.
+     * Its useful when you have input children components
+     * and want to use v-model in the parent component
+     * @param {Event} event
+     * @return {*|default.methods}
+     */
+    propagateValue(event) {
```

Comment:
It should be either `propagate` and `propagateValue` or `propagateEvent` and `propagateEventValue`.

Reply:
updated